biome.json

```diff
@@ -0,0 +1,41 @@
+{
+  "files": {
+    "include":[
+      "static/main/scripts/global/**/*.js",
+      "static/main/scripts/editor/**/*.js"
+    ]
+  },
+  "formatter": {
+    "enabled": true,
+    "formatWithErrors": false,
+    "ignore": [],
+    "attributePosition": "auto",
+    "indentStyle": "space",
+    "indentWidth": 4,
+    "lineWidth": 80,
+    "lineEnding": "lf"
+  },
+  "linter": {
+    "rules": {
+      "style": {
+        "noVar": "off"
```

Comment:
I set [`noVar`](https://biomejs.dev/linter/rules/no-var/) to off because it was wanting any globaly declared variable to be changed to `let` or `const`, however, global variables are required in several places in this project